tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-inventory-tracker-test.php

```
@@ -0,0 +1,512 @@
+		}
+		update_option( WC_Stripe_Agentic_Commerce_Inventory_Tracker::PENDING_UPDATES_OPTION, $pending );
+
+		$extra_product = $this->create_simple_product_with_stock( 99 );
+		$this->sut->track_stock_change( $extra_product );
+
+		$after = get_option( WC_Stripe_Agentic_Commerce_Inventory_Tracker::PENDING_UPDATES_OPTION, [] );
+
+		// Should still be exactly MAX_PENDING_UPDATES, not MAX + 1.
+		$this->assertCount( $max, $after );
+		$this->assertArrayNotHasKey( $extra_product->get_id(), $after );
+	}
```

PHPMD (2.15.0)

[error] The method test_track_stock_change_stops_accumulating_at_threshold is not named in camelCase. (undefined)

(CamelCaseMethodName)